src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -384,6 +399,9 @@ class ProfileCallTreeContextMenu extends PureComponent<Props> {
           Expand all
         </MenuItem>
         <div className="react-contextmenu-separator" />
+        <MenuItem onClick={this.handleClick} data={{ type: 'searchfox' }}>
+          Lookup function on Searchfox
```

Comment:
As text, I'd use: `Look up the function name on Searchfox`. Especially `look up` is 2 words, but also I think `the function name` convey more information about what the action does. What do you think ?

Reply:
Sounds good! The only comment I have, which is really minor, is that perhaps we can drop the `the`? Thus, `Look up function name on Searchfox`. Seems more consistent with `Copy function name`. What do you think @julienw ?

Reply:
I still prefer to keep the "the" in this case; maybe because it's a more complete sentence ? I don't exactly know why I prefer this :)

Reply:
Alright, cool with me :)